src/noise/timers.rs

```diff
@@ -161,6 +173,8 @@ impl Tunn {
         let now = time.duration_since(timers.time_started);
         timers[TimeCurrent].set(now);
 
+        self.update_session_timers(now);
+
         // Load timers only once:
         let session_established = timers[TimeSessionEstablished].time();
         let handshake_started = timers[TimeLastHandshakeStarted].time();
```

Comment:
Maybe fix typos (aut_packet_received / aut_packet_sent) while at it?

Reply:
what is the typo?